content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,24 @@
+title: "A Statement on Richard Stallman and the Free Software Foundation"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+_A Statement from Mixxx on Richard Stallman's return to the FSF._
+
+The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
+His departure in 2019 was a consequence of the criticism he faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks relating to the Epstein scandal.
+His resignation came as a relief for many who have suffered his bad behavior.
+Richard Stallman's actions disqualify him from representing the FSF and the free software community as a whole.
+
+While Mixxx is not officially associated with the Free Software Foundation, there's an inevitable connection in the public's mind between all FOSS projects and the FSF.
+Furthermore, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 or *later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.
+Given this relationship, the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.
+Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation not only of Richard Stallman, but the entire FSF board of directors.
```

Comment:
I think this blog post should be written in the sense of distancing from the FSF and not for calling resignation. For my feeling this was consense among the team. Else we could have sign the open letter. 
```suggestion
```

Reply:
I think it would be negligent to not make calls for specific actions that the FSF could be held accountable for.

Reply:
Or blog ist mit the blace to communicate calls to the FSF. 
In addition it is pretentious to call such things from things based on hearsay.
I don't want to or part of a public storm, fueled by comments on social media.

This public rusch it a strong evidence that the reinstallation of Stallman is read as movement away from inclusion.
Our business is to distancing us from that. 

Calls to the FSF need to be addr seed to them.

Reply:
> Calls to the FSF need to be addressed to them.

Private communications are trivial for them to ignore. They need to be held accountable publicly or nothing will change.

Reply:
No other org has repeated this call on their webside.

Reply:
I have no idea how the FSF's decision to reinstate RMS came to be. I think their decision is wrong, but demanding the whole board of directors to step down - including the female board members who had to put up with Stallman's behavior - is something I don't necessarily agree with.

Reply:
> including the female board members who had to put up with Stallman's behavior

That's beside the point. She already stepped down.

Reply:
Would you agree to calling for Stallman's resignation alone?

Reply:
So you think the board members will read our call on the Mixxx webside and will act on it? 

The only thing this will happen if they actually read it is, they will become angry and will not treat us a serious partner. 

I think the FSF has already proposed future steps l, so this call is outdated any way.

Reply:
> I think the FSF has already proposed future steps l, so this call is outdated any way.

They have not committed to removing all the current board of directors.

Reply:
Doing nothing would be admitting defeat before even trying.

Reply:
> > I think the FSF has already proposed future steps l, so this call is outdated any way.
> 
> They have not committed to removing all the current board of directors.

Who is she? Both Kat Walsh and Odile Bénassy are listed as board members on the FSF's website.

https://www.fsf.org/about/staff-and-board

Reply:
Ah, sorry. Kat Walsh [has resigned](https://fosstodon.org/web/statuses/105948765573856842). I don't know about Odile Bénassy.

Reply:
By allowing Stallman to return, the board of directors is tacitly approving of his behavior and saying it's excusable. I don't see how the board can remain as well. 

What about:
"Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation of Richard Stallman. The FSF board of directors, by allowing him to return, is giving the appearance of endorsing his conduct and it seems doubtful that they can remain effective without significant personnel changes and bylaw reforms."

Reply:
(committed something along those lines)